pkg/network/node/node.go

```diff
@@ -13,9 +13,10 @@ import (
 	"sync"
 	"time"
 
-	log "github.com/golang/glog"
+	"github.com/golang/glog"
```

Comment:
This makes it so that this commit doesn't compile... would be better to put the `log`->`glog` commit first and move this change there.